src/__tests__/konami-code.js

```diff
@@ -0,0 +1,18 @@
+// __tests__/konami-code.js
+// these imports are something you'd normally configure Jest to import for you
+// automatically. Learn more in the setup docs: https://testing-library.com/docs/react-testing-library/setup#cleanup
+import '@testing-library/jest-dom'
+// NOTE: jest-dom adds handy assertions to Jest and is recommended, but not required
+
+import * as React from 'react'
```

Comment:
no longer needed if we update to react 17

Reply:
React isn't a direct dependency of this package, it's a peer dependency, thus I think it's best we keep this in? Will see what I can do in my next iteration when I bring in `tsdx`